source/includes/fact-system-env-vars.rst

```diff
@@ -0,0 +1,81 @@
+.. list-table::
+   :widths: 30 10 40 10 10
+   :stub-columns: 1
+   :header-rows: 1
+
+   * - Environment Variable
+     - Data Type
+     - Description
+     - Default Value (Atlas-hosted)
+     - Default Value (On Premises)
+   * - ``log_level``
+     - integer
+     - Specifies the logging level for |bi-short|:
+
+       * ``-1``: Do not log
+       * ``0``: Log only messages for database users, such as basic
+         ``mongosqld`` events and state changes.
+       * ``1``: Log only messages for ``mongosqld`` administrators.
+       * ``2``: For internal use only.
+     - ``0``
+     - ``2``
+   * - ``max_nested_table_depth``
+     - integer
+     - Specifies the maximum number of unique nested field paths that
+       ``mongosqld`` maps to a relational table for a collection.
+     - ``50``
+     - ``50``
+   * - ``max_num_columns_per_table``
+     - integer
+     - The maximum number of unique fields that ``mongosqld`` maps to
+       relational columns for a collection.
+     - ``1000``
+     - ``2000``
```

Comment:
Where does this `2000` number come from?

Reply:
That's the default value I see on cloud-dev (Atlas-hosted).  Should it be something else?

Reply:
I'm not sure, I'd confirm in #bi_cloud.